### ISSUE TITLE
Preview of CustomTkinter (CTk) not working

### ISSUE THREAD

**Issue by SiL3NC3** [alejandroautalan/pygubu-designer#192]
**Describe the bug**
When starting a Preview (F5), it is not coming up. 

**To Reproduce**
1. Add a "CTk" widget 
2. Press "F5" >> nothing happens
3. Add a "CTkToplevel" widget
4. Press "F5" >> Preview window pops up, as expected

**Expected behavior**
Preview should be working. ;)

**Your Environment (please complete the following information):**
 - OS: Windows 10
 - Python version:  Python 3.9.7
 - Pygubu version: pygubu 0.29
 - Designer version: pygubu-designer 0.34

**Additional context**
Just a hint, add a button to the about dialog to copy out these info for bug reports.
Is always a good feature. ;)

**Comment by alejandroautalan** [comment]
Hello @SiL3NC3, thanks for the report.

It seems to happen on windows OS.  On Linux the screen is shown but with some errors in the background.

F5 preview is trying to execute the real code (and not fake previews like the design window).
The problem here is that Ctk is another Tk  window, with its own TCL interpreter running on background, and some things get not initialized for customtkinter.

I will try to figure out how to display the preview correctly.

Regards

**Comment by alejandroautalan** [comment]
I found a workaround, make sure you have checked "Center toplevel preview window" option in preferences dialog.

![2023-02-18_22-48](https://user-images.githubusercontent.com/3482471/219907945-f2542ed7-f17d-4e77-a3aa-e0f817261595.png)


Note that, if you see the preview,  some issues will still happen because of the problem described  [above](https://github.com/alejandroautalan/pygubu-designer/issues/192#issuecomment-1435380275),
For example, images might not be visible on the preview.

Try it and let me know.

Regards
Alejandro A.

**Comment by SiL3NC3** [comment]
Yes, with the above setting the preview is coming up and images are missing. 
Just a label text is shown.
